var/spack/repos/builtin/packages/py-svgpathtools/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PySvgpathtools(PythonPackage):
+    """A collection of tools for manipulating and analyzing SVG Path objects
+    and Bezier curves."""
+
+    homepage = "https://pypi.org/project/svgpathtools/"
+    url      = "https://pypi.io/packages/source/s/svgpathtools/svgpathtools-1.3.3.tar.gz"
+
+    version('1.3.3', sha256='e4b3784ae41b725fbce6a33a8981210967b16d0b557cb5d98c0ed0c81f0f89b9')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
This is missing dependencies, can you please check https://github.com/mathandy/svgpathtools/blob/fd7348a1dfd88b65ea61da02325c6605aedf8c4f/setup.py#L37 ?